File: fe/lib/models/bai-xe.dart
class BaiXe {
  int? id;
  String? name;
  String? code;
  String? dienTich;
  String? viTri;
  String? slotMax;
  int? status;
  BaiXe({
    this.id,
    this.name,
    this.code,
    this.dienTich,
    this.viTri,
    this.slotMax,
    this.status,
  });
  BaiXe.fromJson(Map<String, dynamic> json)
      : id = json['id'],
        name = json['name'],
        code = json['code'],
        dienTich = json['dienTich'],
        viTri = json['viTri'],
        slotMax = json['slotMax'],
        status = json['status'];
  Map<String, dynamic> toJson() => {
        'name': name,
        'code': code,
        'dienTich': dienTich,
        'viTri': viTri,
        'slotMax': slotMax,
        'status': status,
      };
}

File: BE/src/main/java/com/haivn/common_api/BaiXe.java
package com.haivn.common_api;

import lombok.*;
import org.hibernate.annotations.DynamicUpdate;
import org.hibernate.annotations.Where;

import javax.persistence.*;
@Entity
@Table(name = "bai_xe")
@Getter
@Setter
@DynamicUpdate
@Where(clause = "deleted=false")
public class BaiXe extends BaseEntity{
    @Column(name = "name")
    private String name;
    @Column(name = "code")
    private String code;
    @Column(name = "dien_tich")
    private String dienTich;
    @Column(name = "vi_tri")
    private String viTri;
    @Column(name = "slot_max")
    private String slotMax;
    @Column(name = "status")
    private Short status;

}
